languages/en.ini

```diff
@@ -1415,6 +1415,7 @@ total_resources = "Total Resources"
 total_saved_items = "Total Saved Items"
 total_tags = "Total Tags"
 total_users = "Total Users"
+transit_to = "In transit to %%location%% sent on %%date%%"
```

Comment:
Minor nitpick, but would a comma or semi-colon after `%%location%%` make this a little more readable?